src/Shared/BenchmarkRunner/DefaultCoreConfig.cs

```diff
@@ -30,12 +30,14 @@ public DefaultCoreConfig()
             AddJob(Job.Default
 #if NETCOREAPP2_1
                 .WithToolchain(CsProjCoreToolchain.From(NetCoreAppSettings.NetCoreApp21))
-#elif NETCOREAPP3_0
+#elif NETCOREAPP3_1
                 .WithToolchain(CsProjCoreToolchain.From(new NetCoreAppSettings("netcoreapp3.0", null, ".NET Core 3.0")))
```

Comment:
Please delete these two lines entirely. Mapping `NETCOREAPP3_1` to `netcoreapp3.0` is the **Wrong Thing:tm:** to do.

Reply:
Why was this changed to begin with? @sebastienros should I remove support for 3.0 for the default core config?

Reply:
I suspect the SDK no longer supports `.netcoreapp3.0` but am not sure why that didn't show up elsewhere nor why an `#eliff NETCOREAPP3_0` block would do anything but no-op.